### PR TITLE
TRUS-744 Registration progress with task list

### PR DESCRIPTION
**Pull request by adamconder**
Screenshot 2019-03-21 10 28 17
Screenshot 2019-03-21 10 27 51

### CODE REVIEW

**Comment by manishsw** [comment]
@adamconder  @jassalrichy  Fine for me.

File: test/controllers/TaskListControllerSpec.scala [hunk 1]
```diff
@@ -25,22 +27,146 @@ class TaskListControllerSpec extends SpecBase {
...
+      status(result) mustEqual SEE_OTHER
+
+      redirectLocation(result).value mustEqual routes.TrustHaveAUTRController.onPageLoad(NormalMode).url
 
       application.stop()
     }
+
+    "for an existing trust" when {
+
+      "has matched" when {
```
Comment (jassalrichy) [inline test/controllers/TaskListControllerSpec.scala]: Change to:   "has matched" **must**

File: test/controllers/TaskListControllerSpec.scala [hunk 2]
```diff
@@ -25,22 +27,146 @@ class TaskListControllerSpec extends SpecBase {
...
+
+          contentAsString(result) mustEqual
+            view()(fakeRequest, messages).toString
+
+          application.stop()
+        }
+
+      }
+
+      "has not matched" when {
```
Comment (jassalrichy) [inline test/controllers/TaskListControllerSpec.scala]: Change to: "has not matched" **must**

File: test/controllers/TaskListControllerSpec.scala [hunk 3]
```diff
@@ -25,22 +27,146 @@ class TaskListControllerSpec extends SpecBase {
...
+          status(result) mustEqual SEE_OTHER
+
+          redirectLocation(result).value mustEqual routes.FailedMatchController.onPageLoad().url
+
+          application.stop()
+        }
+
+      }
+
+      "has not attempted matching" when {
```
Comment (jassalrichy) [inline test/controllers/TaskListControllerSpec.scala]: Change to: "has not attempted matching" **must**

File: test/controllers/TaskListControllerSpec.scala [hunk 4]
```diff
@@ -25,22 +27,146 @@ class TaskListControllerSpec extends SpecBase {
...
+          redirectLocation(result).value mustEqual routes.WhatIsTheUTRController.onPageLoad(NormalMode).url
+
+          application.stop()
+        }
+
+      }
+
+    }
+
+    "for a new trust" which {
```
Comment (jassalrichy) [inline test/controllers/TaskListControllerSpec.scala]: Change to: "for a new trust" **must**

File: test/navigation/navigators/MatchingRoutes.scala [hunk 5]
```diff
@@ -81,15 +81,15 @@ trait MatchingRoutes {
 
       "the user does not have a UTR for the trust" when {
```
Comment (jassalrichy) [inline test/navigation/navigators/MatchingRoutes.scala]: Change to: "the user does not have a UTR for the trust" **must**

**Comment by adamconder** [comment]
Test

**Comment by adamconder** [comment]
Testing webhook integration on slack.

**Pull request merged**